intro.ipynb

```diff
@@ -1,1960 +1,1026 @@
 {
```

Comment:
Why did this file get changed?

Reply:
import should change and we make sure it is working with the poetry import?

Reply:
Ah yeah forgot I asked above lol. lgtm